www/www-jekyll.yml

```diff
@@ -0,0 +1,14 @@
+# Install NGINX, and prepare the OME (UoD/SLS) prerequisites
+
+- hosts: www
+  environment:
+      PATH: /usr/local/bin:{{ ansible_env.PATH }}
+
+  roles:
+
+    - role: openmicroscopy.jekyll-build
+      tags: jekyll
```

Comment:
Does the split of the main playbooks into sub-playbooks makes the usage of tags redundant e.g as you should be able to run `www-jekyll.yml` in isolation?

Reply:
Can of worms 😊. Only thing I'd say is if we want to use tags in other playbooks we need to standardise them across the OME, otherwise we'll have problems when trying to use them with something like site.yml.